pkg/api/customization/clusterscan/clusterscan.go

```diff
@@ -33,11 +40,28 @@ func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 			}
 		}
 		if failed {
-			resource.Values["state"] = "error"
+			resource.Values["state"] = errorState
 			return
 		}
 		if completed {
 			resource.Links["report"] = apiContext.URLBuilder.Link("report", resource)
+			cisScanStatusInterface, ok := status["cisScanStatus"]
+			if !ok {
+				resource.Values["state"] = errorState
+				return
+			}
+			cisScanStatus := convert.ToMapInterface(cisScanStatusInterface)
+			failInterface := cisScanStatus["fail"]
+			fail, err := convert.ToNumber(failInterface)
+			if err != nil {
+				logrus.Errorf("error converting failInterface to int: %v", err)
+				resource.Values["state"] = errorState
```

Comment:
should you return after this? if err is != nil likely fail == 0 meaning the the else would mark it passed right?

Reply:
yup, let me fix it.